### PR TITLE
Overview - component update

### PR DESCRIPTION
**Pull request by jerrodvarney**
Major styling updates as well as added functionality for changing the currently selected style and currently selected images. Removed max-width from styles.css to allow for header to expand across screen.

### CODE REVIEW

**Comment by jerrodvarney** [comment]
This pr also contains the refactor of Eriks ratings component that we worked on all day today. I had it as two separate prs but the data was on this branch too, so no need for the second pr.

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -24,20 +24,41 @@ export default function App() {
...
+
+    fetcher.overview.getStylesById(40350)
+      .then(({ data }) => setStyles(data))
+      .catch((err) => console.error('initial style fetch: ', err));
+
+    fetcher.ratings.getReviews(40350)
+      .then(({ data }) => setReviews(data.results))
+      .catch((err) => console.error('initial reviews fetch: ', err));
+
+    fetcher.ratings.getReviewMeta(40350)
```
Comment (boltonlin) [inline client/src/components/App.jsx:45]: Is there a reason for ratings fetcher to be in top-level App component? It seems like the ratings and reviews only get passed down to the Ratings root component, which should be responsible for calling their fetcher. I might be missing something though.

File: client/src/components/overview/sub-comps/Gallery.jsx [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+import {
+  AiOutlineDoubleLeft, AiOutlineDoubleRight, AiOutlineDown, AiOutlineExpand,
+} from 'react-icons/ai';
 
-export default function Gallery(props) {
+export default function Gallery({ currStyle, setCurrStyle }) {
+  // STATE DATA //
+  const [images, setImages] = useState([]);
+  const [heroImage, setHero] = useState('');
+  console.log(images);
```
Comment (boltonlin) [inline client/src/components/overview/sub-comps/Gallery.jsx]:
```suggestion

```
Stray console log

File: client/src/components/ratings/sub-components/ReviewTile.jsx [hunk 3]
```diff
@@ -1,22 +1,18 @@
 import React, { useState, useEffect } from 'react';
 import { AiFillCheckCircle } from 'react-icons/ai';
+import { RxDividerVertical } from 'react-icons/rx';
 import ReviewImageModal from './ReviewImageModal.jsx';
 import fetcher from '../../../fetchers';
+import StarRating from '../../../helpers/star-rating/starRating.jsx';
```
Comment (boltonlin) [inline client/src/components/ratings/sub-components/ReviewTile.jsx]:
```suggestion
import StarRating from '../../../helpers/star-rating/StarRating.jsx';
```
Needs correct file name

**Review by boltonlin — CHANGES_REQUESTED** [review]
Pending meeting. Looks good overall, but I'm not sure why we need to move fetcher for ratings to top level `App.jsx` unless some other component is planning on using it later? I will note that I do not like how this looks currently (see attachment 1) compared to how it used to look (see attachment 2) with the ratings, reviews, bars, and sliders as one column. Also it seems there's not enough space for all 5 stars.

Please ensure that we are able to render the site before making PRs. There's a broken import statement in `ReviewTile.jsx` on `line 6` which makes it so webpack cannot compile. Finally, make sure stray console.logs are cleaned up before making PRs.

chrome_2023-01-31_23-29-23
chrome_2023-01-31_23-30-26

File: client/src/components/ratings/sub-components/ReviewTile.jsx (hunk 3 above)
Comment (eriknewland) [reply client/src/components/ratings/sub-components/ReviewTile.jsx]: Great catch, might explain the warning I got from web-pack too.

**Review by eriknewland — APPROVED** [review]
Looks good on my end, may want to discuss bolton's comments though I believe Jerrod also needs some data from the ratings&review fetcher.

File: client/src/components/App.jsx (hunk 1 above)
Comment (jerrodvarney) [reply client/src/components/App.jsx:45]: Discussed in morning meeting.

**Pull request closed without merging**